src/main/java/org/openmaptiles/util/OmtLanguageUtils.java

```diff
@@ -111,6 +141,8 @@ public static Map<String, Object> getNames(Map<String, Object> tags, Translation
       translations.addTranslations(result, tags);
```

Comment:
Thanks for making the change! This approach looks good, I was thinking though that if we put this same handling directly into `translations.addTranslations` method then we wouldn't need to change planetiler-openmaptiles at all, and other projects would get it for free.